realm/realm-library/src/main/java/io/realm/DynamicRealmObject.java

```diff
@@ -925,6 +926,42 @@ public String toString() {
         return sb.toString();
     }
 
+    /**
+     * Returns {@link RealmResults} containing all {@code srcClassName} class objects that have a relationship
+     * to this object from {@code srcFieldName} field.
+     *
+     * @param srcClassName name of the class returned objects belong to.
+     * @param srcFieldName name of the field to use. Field type must be either {@code io.realm.RealmFieldType.OBJECT} or
```

Comment:
Yes.  I think our docs will be much clearer if we stick to a single set of terms.  The "source class" contains the "source field" . They contain the forward reference.  The "target field" in the "target class" contains the backlink to the source class.